src/test/compile-fail/feature-gate-match_beginning_vert.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
```

Comment:
This needs a `// gate-test-match_beginning_vert` annotation, so the test is counted as a feature gate test.

Reply:
That shouldn't be necessary as long as the test is named correctly, no? That's what tidy said, and most of the other tests ([ex](https://github.com/mattico/rust/blob/f049095809dc124129a9816a1af12080345bd0d6/src/test/compile-fail/feature-gate-simd-ffi.rs)) don't have them.

Reply:
Okay, if this passes testing, then it's good.